examples/usecases/retrieval-with-hyperparameter-optimization.ipynb

```diff
@@ -0,0 +1,1405 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        top_k_rec = model.to_top_k_recommender(item_dataset, 100)
</pre><p>can you pull the latest main branch and  check again if this line is still working? <code>model.to_top_k_recommender(item_dataset, 100)</code>  this  <a href="item_dataset, 100" target="_blank">PR</a> merged this morning, and this might  cause some changes in top_k_recommender. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/786/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
thank you for highlighting this, just checked, everything still works